depends/packages/zeromq.mk

```diff
@@ -17,7 +17,9 @@ endef
 
 define $(package)_preprocess_cmds
   patch -p1 < $($(package)_patch_dir)/remove_libstd_link.patch && \
-  cp -f $(BASEDIR)/config.guess $(BASEDIR)/config.sub config
+  patch -p1 < $($(package)_patch_dir)/fix_dash_compat.patch && \
+  cp -f $(BASEDIR)/config.guess $(BASEDIR)/config.sub config && \
+  ./autogen.sh
```

Comment:
Why `./autogen.sh` is a part of `$(package)_preprocess_cmds`? On the master branch it belongs to `$(package)_config_cmds`:https://github.com/bitcoin/bitcoin/blob/3ce40e64d4ae9419658555fd1fb250b93f52684a/depends/packages/zeromq.mk#L27-L30

Reply:
Changed.